app/controllers/ActivitiesController.scala

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import com.google.inject.Inject
+import play.api.i18n.I18nSupport
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import services.{ActivitiesService}
+import uk.gov.hmrc.domain.Nino
+import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendBaseController
+import views.html.ActivitiesView
+
+import scala.concurrent.ExecutionContext
+
+class ActivitiesController @Inject()(
+                                      val controllerComponents: MessagesControllerComponents,
+                                      val activitiesService: ActivitiesService,
+                                      view: ActivitiesView
+                                           )(implicit executionContext: ExecutionContext)
+  extends FrontendBaseController with I18nSupport {
+
+  def onPageLoad: Action[AnyContent] = Action.async { implicit request =>
+    activitiesService.getActivities(Nino("GG012345C")).map { activities =>
```

Comment:
this method needs authentication otherwise it's accessible without logging in:   `def onPageLoad: Action[AnyContent] = (authenticate andThen getUserDetails) { implicit request =>`

Also, hardcoding the NINO is fine but the code for collecting the NINO from Auth needs to be in place first: 
`request.authenticatedRequest.nino` then just ignore that and hardcode as you're doing